### PR TITLE
Added comparison and repository creation for stats

### PR DESCRIPTION
**Pull request by vyper**
Criar um service object:
- [x] retornar resultado para view
- [x] persistir (async) repositórios
- [x] persistir (async) comparação
- [x] incrementar contador de comparações

### CODE REVIEW

File: app/models/repository.rb [hunk 1]
```diff
@@ -1,31 +1,16 @@
...
   #
   # attributes
   #
-  ATTRIBUTES = %i(name description created_at pushed_at homepage
+  ATTRIBUTES = %i(description created_at pushed_at homepage
                   language stargazers_count forks_count open_issues_count
                   network_count subscribers_count watchers_count)
 
-  attr_accessor :repository
+  attr_accessor *ATTRIBUTES
```
Comment (vyper) [inline app/models/repository.rb]: Podia mudar pra `GITHUB_ATTRIBUTES`. HEHE

Comment (neves) [inline app/models/repository.rb]: Let's see how to handle other sources together like rubygems or stackoverflow

**Comment by neves** [comment]
Let's put this in hold till the end of Rails Rumble.

File: app/models/repository.rb (hunk 1 above)
Comment (vyper) [inline app/models/repository.rb]: Good point.

File: app/views/compare/_repo.html.slim [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .media
   .media-left
-    = image_tag repo.owner.avatar_url, width: 100, height: 100, class: 'media-object'
+    // TODO: repo.owner is a attribute for repository model
```
Comment (vyper) [inline app/views/compare/_repo.html.slim]: @neves maybe we need a decorator or presenter here... many fields conflicting between model and github response. );

**Comment by vyper** [comment]
@neves I waiting for response about decorator/presenter and your code review. But, it's done.

File: app/views/compare/_repo.html.slim (hunk 2 above)
Comment (neves) [inline app/views/compare/_repo.html.slim]: Agree!

**Comment by neves** [comment]
:+1:

**Pull request merged**